.github/workflows/keiko_test.yml

```diff
@@ -44,6 +44,11 @@ jobs:
         run: pip install -r requirements-dev.txt
         working-directory: ./keiko
 
+      - name: Create writeable directory for reports (edge case for GitHub Actions)
+        run: mkdir -p /tmp/keiko/reports && chmod -R 700 /tmp/keiko/reports
+
       - name: Run unit tests
         run: make test
         working-directory: ./keiko
+        env:
+          KEIKO_REPORTS_FOLDER: /tmp/keiko/reports
```

Comment:
Missing newline at the and of the file. pre-commit didn't fix this because of #1660

Reply:
1660 is now merged.